### ISSUE TITLE
[Suggestion] Keybindings For Disambiguated Paste Options

### ISSUE THREAD

**Issue by Krissalyce** [JannisX11/blockbench#1149]
Hello,

This is a feature request for disambiguated paste options as hotkeys. Disambiguated paste options, like paste face geometry, paste UVs, etc. would be very useful as hotkeys in order to improve productivity for advanced users.

The current method of paste is perfect. It is easy to use and understand. Ease of use is a fundamental part of Blockbench that is important to preserve. By keeping the current paste option the default, this would be ideal for new users.

However, as an advanced user with several custom hotkeys, it can be cumbersome to confirm every paste with a mouse-click, and slows down productivity. Since I am already familliar with the Blockbench workflow, it would be useful to have the disambiguous actions of "paste uv, paste geometry" etc. as hot-keyable actions. The default for these actions should be unbound, to prevent accidental user error, but can be bound by advanced users by editing the keymapping.

The sentiment that this would improve ease of use and productivity is shared by other members. See the following [Discord message.](https://discord.com/channels/314078526104141834/314078934155132948/903313356264468510) Having to click once or twice is no pain, but in a larger project, this can quickly add up to hundreds of clicks, delaying productivity.

In the past, it was recommended that the context menu be hidden behind a hotkey - this suggestion is similar but the opposite. Keep the ambiguous paste with context menu as default, and add disambiguous paste as an optional keybinding for advanced users.

In conclusion, adding disambiguated paste would greatly benefit advanced users, for the additional simplicity, usability, and productivity it would bring.

Thank you for your consideration.
